### ISSUE TITLE
Windows: ModuleNotFoundError: No module named 'win32api'

### ISSUE THREAD

**Issue by ilyasher** [NVIDIA/cuda-python#32]
Installing on Windows:

`python -m pip install cuda-python`

Then from python:

`from cuda import cuda`

Fails with

```
    File "cuda\cuda.pyx", line 1, in init cuda.cuda

    File "cuda\ccuda.pyx", line 1, in init cuda.ccuda

    File "cuda\_cuda\ccuda.pyx", line 8, in init cuda._cuda.ccuda

  ModuleNotFoundError: No module named 'win32api'
```

I can fix this by installing `pypiwin32` manually. But I think it should be listed in requirements.txt if platform_system is Windows.

Thanks

**Comment by leofang** [comment]
Sorry for lack of response. I actually am puzzled too because IIRC at one point we did have `pywin32` listed but it seems to be gone now. @vzhurba01 do you know what happened?

**Comment by vzhurba01** [comment]
I don't think the file had called it out explicitly before. I've now added it to requirements.txt if platform_system is Windows. This change will get picked up in the next wheel release.

Leaving the bug open until the next set of wheels are released and the fix is confirmed.

**Comment by vzhurba01** [comment]
Fixed with [12.6](https://github.com/NVIDIA/cuda-python/releases/tag/v12.6.0) release, closing.
